index.html

```diff
@@ -89,89 +94,153 @@
+  var binCounty = 15
   // FETCH DATA
-
   d3.json('data.json', function (error, json) {
     if (error) { throw error; }
-
     data = json;
-
-    setup();
-    update();
+    var chart = new Chart();
   });
+      function Chart() {
+        var chart = this;
 
-  function setup() {
-    // SVG
-
-    svg = d3.select('#chart')
+        chart.svg = d3.select('#chart1')
       .append('svg')
       .attr('width', width + margin.right + margin.left)
       .attr('height', height + margin.top + margin.bottom)
       .append('g')
       .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
-
     // SCALES
-
-    x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+    chart.x = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.year; }))
       .range([0, width])
       .nice();
-
-    y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+    chart.y = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.value; }))
       .range([height, 0])
       .nice();
-
+    
     // AXES
-
-    // EVENT HANDLERS
-
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
+    // format x axis
+    var formatyears = d3.format("1000")
+    // format y axis
+    var formatmoney = d3.format("$1,000,000,000")
+    //Define X axis
+   
+    var xAxis = d3.svg.axis()
+             .scale(x)
+             .orient("bottom")
+             .ticks(20)
+             .tickSize(height + 15)
+             .tickFormat(formatyears);
+    //Define Y axis
+    var yAxis = d3.svg.axis()
+             .scale(y)
+             .tickSize(width)
+             .tickFormat(formatmoney)
+             .orient("right")
+             .ticks(5);
+   
+    //Create X axis
+    var chart.gx = svg.append("g")
+          .attr("class", "axis")
+          .attr("transform", "translate(0," + height + ")")
+          .append("text")
+          .attr("x", width)
+          .attr("y", 0)
+          .style("text-anchor", "end")
+          .text("Year");
+    //Create y axis
+  var chart.gy = svg.append("g")
+      .attr("transform", "rotate(-90)")
+      .attr("class", "axis")
+      .append("text")
+      .style("text-anchor", "end")
+      .text("Incentive Value") 
+      .attr("x", -10)
+      .attr("y", -10);
+    var chart.gy = svg.append("g")
+      .attr("class", "y axis")
+      .call(yAxis);
+    chart.gy.selectAll("g").filter(function(d) { return d; })
+      .classed("minor", true);
+    
+    var chart.gx = svg.append("g")
+      .attr("class", "x axis")
+      .call(xAxis);
+      
+    chart.gx.select('g').filter(function(d) { return d; })
+        .classed("minor", true);
+    chart.gy.selectAll("text")
+    .attr("x", 4)
+    .attr("dy", -4);
+      // EVENT HANDLERS
+    d3.select('#location').on('change', function () {
+      options.location = d3.event.target.value;
       update();
     });
```

Comment:
update() isn't a function anymore. You need to install call the update method of the chart constructor function.